wallet-new/src/Cardano/Wallet/API/V1/Types.hs

```diff
@@ -305,9 +307,26 @@ instance Arbitrary Wallet where
                      <*> pure "My wallet"
                      <*> arbitrary
 
+--------------------------------------------------------------------------------
+-- Addresses
+--------------------------------------------------------------------------------
+
+-- | Whether an address is valid or not.
+newtype AddressValidity = AddressValidity { getValid :: Bool }
+  deriving (Eq, Show, Generic)
+
+deriveJSON Serokell.defaultOptions ''AddressValidity
```

Comment:
I know we discussed this previously, but I still think renaming this `isValid` makes much more sense than adhering blindly to code standard. I will take the bullet for the rest of the team 😅

Reply:
Haha, okay I will change it in this case.